tests/ablateLibrary/chemistry/chemTabModelTests.cpp

```diff
@@ -70,12 +73,7 @@ TEST_P(ChemTabModelGetSpeciesAndProgressVariableTestFixture, ShouldReturnCorrect
 
 INSTANTIATE_TEST_SUITE_P(ChemTabModelTests, ChemTabModelGetSpeciesAndProgressVariableTestFixture,
                          testing::Values((ChemTabModelGetSpeciesAndProgressVariableTestParameters){
-                             .modelPath = "inputs/chemistry/chemTabTestModel_1",
-                             .expectedSpecies = {"YiCH4",   "YiH",     "YiO",    "YiO2",   "YiOH",    "YiH2O",  "YiHO2",   "YiH2O2",   "YiC",      "YiCH",   "YiCH2",  "YiCH2(S)", "YiCH3",  "YiH2",
-                                                 "YiCO",    "YiCO2",   "YiHCO",  "YiCH2O", "YiCH2OH", "YiCH3O", "YiCH3OH", "YiC2H",    "YiC2H2",   "YiC2H3", "YiC2H4", "YiC2H5",   "YiC2H6", "YiHCCO",
-                                                 "YiCH2CO", "YiHCCOH", "YiN",    "YiNH",   "YiNH2",   "YiNH3",  "YiNNH",   "YiNO",     "YiNO2",    "YiN2O",  "YiHNO",  "YiCN",     "YiHCN",  "YiH2CN",
-                                                 "YiHCNN",  "YiHCNO",  "YiHOCN", "YiHNCO", "YiNCO",   "YiC3H7", "YiC3H8",  "YiCH2CHO", "YiCH3CHO", "YiN2",   "YiAR"},
-                             .expectedProgressVariables = {"zmix", "CPV_0", "CPV_1", "CPV_2", "CPV_3"}}));
+                             .modelPath = "inputs/chemistry/chemTabTestModel_1", .expectedSpecies = {SPECIES_NAMES}, .expectedProgressVariables = {CPV_NAMES}}));
```

Comment:
It looks like these tests are setup to run multiple themTabTestModels so hardcoding having a single set of SPECIES_NAMES,  CPV_NAMES, etc won't allow for testing multiple models. If we want to have a file containing test results I think a text file like yaml (which can still be auto generated)that is parsed by the test code might work better.  There is already a yaml parser in the code.  That way we could store it in the model directory but pass the path in the test params like the model.

Reply:
Yes that is why I created the adapt_model_tests.py script, it generates a header file with macros for the test targets. The header file is then included inside the cpp test file & used for comparison.

Reply:
It doesn't work very well as automated testing parameters. These are automated test that run in ablate, not something that you modify each time.  ChemTabModelTests is parameterized so that they can run multiple sets of input/outputs without recompiling or regenerating the tests.  This can't be done with a single header file with non-specific defines. In the future we may also want to use an ablate library executable to test the models in chemTabRepo we are going to populate, we can't do this with compile time constants.  In order for this to be implemented in ablate, please change the tests to allow for multiple parameter to be tested.  If you want to use autogenerated test inputs/outputs please use runtime parameters such as an yaml file.

Reply:
You don't need to worry about how the header file is generated.
What I'm doing is _exactly what Varun's already merged code does_ except now: it is more organized,
a bunch of related constants are grouped together in one header file (rather than baked inline) &
it has a much better model than the previous one.

I think we should prioritize actually testing the model.
But if _you_ really want to add yaml files, you can feel free.